frontend/src/components/update-keypair-dialog/update-keypair-dialog.tsx

```diff
@@ -0,0 +1,269 @@
+import { useCallback } from 'react'
```

Comment:
i did indeed split it in two pieces (took out the `hooks/use-keypair-update` part), but i'd like to keep this PR in the realms of the UI changes, instead of over-optimizing every file.